### DISCUSSION TITLE
I'd love to be able to add timers that already happened

### DISCUSSION THREAD

**Discussion by kenjibailly** [mcndt/obsidian-toggl-integration#57] [Feature Requests]
Sometimes I have something that I forgot to track, or a custom input that I couldn't track at that moment. 
Would be nice to be able to add custom length timers that already happened. 
Right now I only see the option to add a timer and track its time.

**Comment by PunchArm255** [comment]
Up.

**Comment by mcndt** [comment]
Hi everyone, thanks for the feedback.

In all honesty, this is probably not something I can implement soon. If I have more time in the summer, I might get back to this feature request, but I'm currently only able to spend time maintaining existing features (i.e. the reporting features).

**Comment by mcndt** [comment]
Added to issue tracker #58
